Automattic/wp-calypso
# WooCommerce: Remove old action-header

When action-header was put into its own directory, a rebase/merge
brought back the old one. No code should be referencing this, so it
should make no visible difference.

The file that replaces this one is `client/extensions/woocommerce/components/action-header/index.js`

To Test:

`make run`
Visit `http://calypso.localhost:3000/store/products/<slug>/add`
Verify the header displays correctly and buttons work.
